symforce/codegen/backends/cpp/cpp_config.py

```diff
@@ -72,3 +72,7 @@ def printer(self) -> CodePrinter:
     @staticmethod
     def format_data_accessor(prefix: str, index: int) -> str:
         return f"{prefix}.Data()[{index}]"
+
+    @staticmethod
+    def format_matrix_accessor(key: str, i: int, j: int, *, shape: T.Tuple[int, int]) -> str:
```

Comment:
Don't think `shape` needs to be keyword-only if it's a different type from everything else, but fine either way